src/frontend/src/components/TimeSlot.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box } from '@mui/system';
+import { getIcon } from '../utils/design-review.utils';
+
+interface TimeSlotProps {
+  text?: string;
+  fontSize?: string;
+  backgroundColor?: string;
+  icon?: string;
+  isModal: boolean;
+  onMouseDown?: (e: any) => void;
+  onMouseEnter?: (e: any) => void;
+  onMouseUp?: (e: any) => void;
+  onMouseOver?: () => void;
+}
+
+const TimeSlot: React.FC<TimeSlotProps> = ({
+  text,
+  fontSize,
+  backgroundColor,
+  isModal,
```

Comment:
you can also say isModal = 'false' here, so it's by default false. That way you won't have to even add that prop in anywhere isModal is false